uswds/uswds
# USWDS - Icon: Updated the Twitter/X logo to meet new brand guidelines.

# Summary

Updated the Twitter/X logo to meet the new [brand specifications](https://about.twitter.com/en/who-we-are/brand-toolkit)

## Breaking change

This is not a breaking change.

## Related issue

Closes #5471

## Related pull requests

There are no related pull requests.

## Preview link

A preview of the updated svg is visible in this pull request's commit log.

## Problem statement

In July, Twitter/X updated their logo and associated brand toolkit. The current uswds distribution uses the old "bird" logo.

## Solution

This PR replaces the old "bird" logo with the new "X" logo.

## Testing and review

In a variety of browsers:

- Confirm that the logo renders as expected:
  - Using different backgrounds and fills
  - When scaled
- Confirm that the provided svg meets Twitter's brand standards.
